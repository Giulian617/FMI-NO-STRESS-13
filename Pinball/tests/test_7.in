9 1
8 1 2
3 8 10
9 8 6
4 8 9
6 4 6
5 3 10
7 5 10
2 6 3
