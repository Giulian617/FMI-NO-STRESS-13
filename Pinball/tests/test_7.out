265210086
933445385
914285721
887058830
